tasks/databases.yml

```diff
@@ -14,7 +14,7 @@
     lc_ctype: "{{ item.lc_ctype | default(postgresql_ctype) }}"
     port: "{{postgresql_port}}"
     template: "template0"
-    state: present
+    state: "{{ item.state | default(present)"
```

Comment:
This is a syntax error, missing the closing `}}`. right?  Can you please get it to compile and pass the automated tests.

Reply:
thank you! I was looking at this thing for ages trying to figure out where my syntax error was. Figures it was right in front of my face.